### DISCUSSION TITLE
Focusing a disabled Tab

### DISCUSSION THREAD

**Discussion by hrigner** [radix-ui/primitives#3022] [Ideas]
Hey!

I think a disabled `Tab` should be focusable, reading the [a11y recommendations](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#focusabilityofdisabledcontrols). This is however not possible with the radix-ui tabs.. even if I set `tabIndex` to 0, the arrow keys are not possible to use for focusing the disabled tab. WDYT?

**Comment by 58bits** [comment]
We've just come across this - and agreed.

**Comment by 58bits** [comment]
Okay have also seen that you can use the left and right arrows to move between all tabs. Does this satisfy accessibility requirements?
